src/backend/cgobj.c

```diff
@@ -3702,5 +3708,94 @@ void Obj::gotref(Symbol *s)
 {
 }
 
+/*****************************************
+ * write a reference to a mutable pointer into the object file
+ * Input:
```

Comment:
This applies throughout the PR. Also, `symbols'` should be `Symbol's` globally to this PR.